packages/spawner/package.json

```diff
@@ -33,7 +33,7 @@
     "@agoric/harden": "^0.0.4",
     "@agoric/insist": "^0.0.1",
     "@agoric/make-promise": "^0.0.1",
-    "@agoric/nat": "^3.0.1",
+    "@agoric/nat": "^2.0.1",
```

Comment:
that's weird that `yarn workspaces info` didn't catch this

Reply:
ohh, maybe it's because `@agoric/Nat` isn't defined in the monorepo, it's external

Reply:
Yeah, I think that's why